registries/adapters/apiv2_adapter_test.go

```diff
@@ -154,7 +153,9 @@ func TestAPIV2GetImageNames(t *testing.T) {
 func TestAPIV2GetImageNamesBadURL(t *testing.T) {
 	// This test also covers the test case of a registry that does not implement discovery
 	// AKA <url>/v2/_catalog returns 404
-	testConfig.URL, _ = url.Parse("https://www.google.com")
+	serv := adaptertest.GetAPIV2Server(t, false)
```

Comment:
This is better because we are using a real APIv2 endpoint that simply doesn't have /v2/_catalog